bin/scripts/config-helper.ts

```diff
@@ -0,0 +1,157 @@
+    },
+    {
+      type: "input",
+      name: "AWS_REGION",
+      message: "The region that MLSpace resources will be deployed into.",
+    },
+    {
+        type: "input",
+        name: "OIDC_URL",
+        message: "The OIDC endpoint that will be used for MLSpace authentication",
+    },
+    {
+        type: "input",
+        name: "OIDC_CLIENT_NAME",
+        message: "The OIDC client name that should be used by MLSpace for authentication",
+    },
+]
+
+
+console.log(figlet.textSync("MLSpace Config Wizard"));
+
+createConfig();
+
+async function createConfig() {
+    const basic = "basic";
+    const advanced = "advanced";
+    const advancedOrBasicResponse = await prompt({
+        type: "select",
+        name: "configType",
+        choices: [
+            { message: "Basic Config - only prompts for the properties which must be set in order to deploy MLSpace.", name: basic },
+            { message: "Advanced Config - prompts for required fields as well as optional configurations which are commonly customized.", name: advanced },
+        ],
+        message: "Select a configuration type:",
+    })
+
+    if(advancedOrBasicResponse.configType === advanced) {
+        await advancedPrompts();
+        console.log(questions.length)
+    }
+
+
+    // The final list of questions has been created, this answer object will be the final config JSON
+    const answers = await prompt(questions);
+    
+      
+    console.log(`Response: ${JSON.stringify(answers)}`); 
+
+    fs.writeFileSync('./lib/config.json', JSON.stringify(answers, undefined, 4));
+}
+
+
+async function advancedPrompts() {
+    const vpcResponse = await prompt({
+        type: "confirm",
+        name: "existingVpc",
+        message: "Do you want to use existing VPC? (selecting false will create a new VPC for MLSpace)",
+    })
+    if(vpcResponse.existingVpc) {
+        const vpcQuestions = [
+            {
+                type: "input",
+                name: "EXISTING_VPC_NAME",
+                message: "If MLSpace is being deployed into an existing VPC this should be the name of that VPC",
+            },
+            {
+                type: "input",
+                name: "EXISTING_VPC_ID",
+                message: "If MLSpace is being deployed into an existing VPC this should be the ID of that VPC",
+            },
+            {
+                type: "input",
+                name: "EXISTING_VPC_DEFAULT_SECURITY_GROUP",
+                message: "If MLSpace is being deployed into an existing VPC this should be the default security group of that VPC",
+            },
+        ]
+        questions.push(...vpcQuestions);
+    }
+
+    const roleResponse = await prompt({
+        type: "confirm",
+        name: "existingRoles",
+        message: "Do you want to use existing IAM Roles? (selecting false will create new IAM Roles for MLSpace actions)",
+    });
+    if(roleResponse.existingRoles) {
+        const roleQuestions = [
```

Comment:
Is it possible to leave some of these blank to generate default roles while providing others?

Reply:
Yes if you leave some blank and fill in others, it will only use the ones which were filled in.